File: src/main/java/com/cuupa/mailprocessor/userconfiguration/ConfigurationProperties.java
package com.cuupa.mailprocessor.userconfiguration;

import com.google.common.base.Strings;
import org.apache.commons.lang3.StringUtils;

import java.io.Serializable;

public class ConfigurationProperties implements Serializable {

    protected String getPasswordStared(final String password) {
        if (StringUtils.isEmpty(password)) {
            return "";
        }
        return Strings.repeat("*", password.toCharArray().length);
    }
}

File: src/main/java/com/cuupa/mailprocessor/userconfiguration/ConfigurationProperties.kt
package com.cuupa.mailprocessor.userconfiguration

import com.google.common.base.Strings
import java.io.Serializable

open class ConfigurationProperties : Serializable {

    protected fun getPasswordStared(password: String?): String {
        return if (password.isNullOrEmpty()) {
            ""
        } else Strings.repeat("*", password.toCharArray().size)
    }
}